### PR TITLE
fix #64912 my.bookmark.rakuten.co.jp

### LINKED ISSUE #64912: my.bookmark.rakuten.co.jp
### Issue URL (Anti Adblock Script)
 [https://my.bookmark.rakuten.co.jp/shop/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fmy.bookmark.rakuten.co.jp%2Fshop%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Safari v1.8.2
Browser: | Safari
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Other:</b><br/> Filter unblocking search ads and self-promotion<br/><br/><b>Language-specific:</b><br/> AdGuard Japanese

### PR DESCRIPTION
**Pull request by Yuki2718**
https://github.com/AdguardTeam/AdguardFilters/issues/64912
Account and bookmarking a store required to reproduce, probably hard for you. I'm not sure why `ranking.rakuten.co.jp` is blocked though.

### CODE REVIEW

File: JapaneseFilter/sections/whitelist.txt [hunk 1]
```diff
@@ -1,6 +1,8 @@
 !
 ! White list. Fixing filtration errors(false-positive)
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/64912
+@@||ranking.rakuten.co.jp^$domain=bookmark.rakuten.co.jp
```
Comment (Alex-302) [inline JapaneseFilter/sections/whitelist.txt:5]: I see in AdAway:
`127.0.0.1 api.ranking.rakuten.co.jp`
`127.0.0.1 shopapi.ranking.rakuten.co.jp`
Which URL is blocked?

Comment (Yuki2718) [reply JapaneseFilter/sections/whitelist.txt:5]: The latter:

rakuten2

I have never used AdAway.

Comment (Yuki2718) [reply JapaneseFilter/sections/whitelist.txt:5]: You beat me, but can you add
`||rakuten.co.jp/com/img/thumb/logout/bnr/`?

For this:

rakuten

Comment (Alex-302) [inline JapaneseFilter/sections/whitelist.txt:5]: Ok, removed blocking rule.

Comment (Yuki2718) [reply JapaneseFilter/sections/whitelist.txt:5]: Thx!

**Pull request closed without merging**